src/pages/Login.jsx

```diff
@@ -7,15 +7,20 @@ import "../styles/Login.css";
 import { Link, useNavigate } from "react-router-dom";
 import {loginInputNames, loginInputsInitialValues} from "../utils/constants.js";
 import { loginSchema } from "../utils/formSchema.js";
-import { useAppProvider } from "../context/AppContext.js";
+import { useDispatch, useSelector } from "react-redux";
+import { loginSignUpActions as loginActions } from "../store/login-signUp-slice.js";
+import { sessionsActions } from "../store/sessions-slice.js";
 
 const Login  = () => {
-    const { showLoginErrorMsg, setShowLoginErrorMsg, updateIsAUserLogged } = useAppProvider();
     const navigateTo = useNavigate();
-
+    const dispatch = useDispatch();
+    const loginErrors = useSelector(state => state.loginSignUp.loginErrors);
     useEffect(() => {
-        setShowLoginErrorMsg({ show: false, msg: ""});
-    }, [setShowLoginErrorMsg]);
+        dispatch(loginActions.setLoginErrors({
+            show: false,
+            msg: ""
+        }));
+    }, [dispatch]);
```

Comment:
there's no need to do this, let's set those values as the default ones in the slice files and that's it